### ISSUE TITLE
wrong-number-of-arguments error in Emacs 25.3 and older

### ISSUE THREAD

**Issue by hlissner** [alexmurray/ivy-xref#14]
```
(wrong-number-of-arguments (3 . 3) 2)
```

This is caused by #11. Specifically, the signature of `xref--show-pos-in-buf` has changed between Emacs 25.3 and 26.1.

```elisp
;; 25.3
(xref--show-pos-in-buf POS BUF SELECT)
;; 26.1
(xref--show-pos-in-buf POS BUF)  ; what #11 is using
```

I'd suggest either bumping the `(emacs "25.1")` dependency to `(emacs "26.1")`, or conditionally use one or the other function depending on the current version of Emacs.

**Comment by hlissner** [comment]
Or, fall through to the other function if the first one fails:

```elisp
(condition-case _
    (xref--show-pos-in-buf POS BUF)
  (wrong-number-of-arguments (xref--show-pos-in-buf POS BUF t)))
```

**Comment by alexmurray** [comment]
Thanks for creating this issue and for the proposed fix in #15 - but I prefer the more explicit approach - so have just committed https://github.com/alexmurray/ivy-xref/commit/f8ab1eeab05f025a97ddf2f239862db14e3430e9 - let me know if this does not resolve the issue for you.
